public/app/assetlibrary/assetlibrary_addlink/assetlibrary_addlink.html

```diff
@@ -0,0 +1,24 @@
+<div data-ng-controller="AssetLibraryAddLinkController">
+
+  <h1>Add a new link</h1>
+
+  <form name="assetlibraryAddlinkForm" ng-submit="assetlibraryAddlinkForm.$valid && addLink()" novalidate>
```

Comment:
@nicolaasmatthijs seems like we're mixing `data-ng-` and `ng-`. Either one works fine but should we try to be consistent?